File: SDKs/noyo.com/1.0/go/pkg/models/operations/getagentlocation.go
package operations

import (
	"openapi/pkg/models/shared"
)

type GetAgentLocationPathParams struct {
	AgentID    string `pathParam:"style=simple,explode=false,name=agent_id"`
	LocationID string `pathParam:"style=simple,explode=false,name=location_id"`
}

type GetAgentLocationRequest struct {
	PathParams GetAgentLocationPathParams
}

type GetAgentLocationResponse struct {
	AgentLocationResult *shared.AgentLocationResult
	ContentType         string
	StatusCode          int64
}

File: SDKs/noyo.com/1.0/typescript/src/sdk/models/operations/getagentlocation.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class GetAgentLocationPathParams extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=agent_id" })
  agentId: string;

  @SpeakeasyMetadata({ data: "pathParam, style=simple;explode=false;name=location_id" })
  locationId: string;
}


export class GetAgentLocationRequest extends SpeakeasyBase {
  @SpeakeasyMetadata()
  pathParams: GetAgentLocationPathParams;
}


export class GetAgentLocationResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  agentLocationResult?: shared.AgentLocationResult;

  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  statusCode: number;
}
